web-client/public/js/grnstate.js

```diff
@@ -46,9 +49,7 @@ export const grnState = {
         this.newNetwork = true;
     },
 
-    showWeightsMouseover: true,
-    showAllWeights: false,
-    hideAllWeights: false,
+    edgeWeightDisplayOption: showWeightsMouseover,
```

Comment:
For example here, just reading this line you're not sure if `showWeightsMouseover` is a variable or a constant. Making it all caps like `SHOW_WEIGHTS_MOUSEOVER` makes this clearer without needing a color-coding editor.

Make sure to expand your awareness of the code toward any conventions that exist. If something is being done in the code for no clear reason, it is worthwhile to ask.

When I work with other programmers, a positive sign (for me) is when they notice the _non_-existence of coding conventions. It means that they are actively looking at the source code and recognizing patterns in style and organization. And they can then tell when such patterns are insufficient or non-existent.